packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberbox.localization.tests.js

```diff
@@ -0,0 +1,76 @@
+import $ from 'jquery';
+import localization from 'localization';
+
+import 'generic_light.css!';
+import keyboardMock from '../../helpers/keyboardMock.js';
+
+import 'ui/number_box';
+import 'ui/validator';
+import 'ui/text_box/ui.text_editor';
+
+QUnit.testStart(function() {
+    const markup =
+        '<div id="qunit-fixture">\
+            <div id="numberbox"></div>\
+            <div id="widget"></div>\
+            <div id="widthRootStyle"></div>\
+        </div>';
+
+    $('#qunit-fixture').html(markup);
+    $('#widthRootStyle').css('width', '300px');
+});
+
+const moduleConfig = {
+    beforeEach: function() {
+        this.$element = $('#numberbox').dxNumberBox({
+            format: '#0.##',
+            value: '',
+            useMaskBehavior: true
+        });
+        this.clock = sinon.useFakeTimers();
+        this.input = this.$element.find('.dx-texteditor-input');
```

Comment:
It's better to store class names at the top of the file inside constant variables.